bundles/core/src/test/resources/link/exporter-linkcomponent-v2.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id"        : "link-b335f8f10c",
+    "title"     : "Link Component",
+    "tooltip"   : "Some Hover Tooltip Text",
+    "assetType" : "ADAPTIVE_FORM",
+    "assetPath" : "/content/dam/formsanddocuments/sample-form",
+    "accessibilityLabel"       : "Test Aria Label",
```

Comment:
Shouldn't by default title be part of aria-label ? Do we need a separate property for this ?

Reply:
I checked the implementation of LinkHandler, and they are not setting aria-label if not explicitly set. For e.g the links used in Teaser don't have the aria-label set, but the links used in Download component have an aria-label set with an autogenerated value from the asset to be downloaded.

From the w3c spec on this attribute, I found that aria-label is meant to be used on an element whose visible value is not same as it's semantic value [1]. E.g a link to close a modal has a text `X` but actually represents the action `close`. Hence I've kept it as optional and am setting it only if explicitly input by the author. 

Should we instead put an auto-generated value / keep it same as tooltip / link text?

[1]: https://www.w3.org/TR/wai-aria/#aria-label